.docker/web/Dockerfile

```
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1.4
+FROM python:3.11-slim-bullseye AS builder
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)